ktlint-ruleset-template/build.gradle

```diff
@@ -32,16 +32,16 @@ dependencies {
 
   testImplementation projects.ktlintCore
   testImplementation projects.ktlintTest
-  testImplementation deps.junit5
-  testImplementation deps.assertj
+  testImplementation libs.junit5
+  testImplementation libs.assertj
 }
 
 test {
   useJUnitPlatform()
 }
 
-task ktlint(type: JavaExec, dependsOn: classes) {
-  main = 'com.pinterest.ktlint.Main'
+task ktlint(type: JavaExec, dependsOn: classes, group: LifecycleBasePlugin.VERIFICATION_GROUP) {
```

Comment:
Solve `ktlint` task doesn't appear in `verification` group, link https://github.com/pinterest/ktlint/pull/1316#issuecomment-1019443372.